### ISSUE TITLE
Internal server error occurs 

### ISSUE THREAD

**Issue by tsh-box** [me-box/databox#292]
Hello,
when I built the latest Databox from me-box,the following errors are occuring 
V0.5.0 is not work well.

container-manager       : ｫ2018-10-15T04:26:34.047760516Z [ERROR]2018/10/15 04:26:34 /containerManager.go L159:[GetAllSLAs] failed to get  slaStorecontainer-manager    : . Error getting latest data: internal server error
ｫ2018-10-15T04:26:34.047771717Z [ERROR]2018/10/15 04:26:34 /containerMcontainer-manager : anager.go L159:[GetAllSLAs] failed to get  slaStore. Error getting latest data: internal server error
container-manager       : 2018-10-15T04:26:34.585194445Z [ERROR]2018/10/15 04:26:34 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:34.702591840Z [ERROR]2018/10/15 04:26:34 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:35.661578419Z [ERROR]2018/10/15 04:26:35 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:36.750362829Z [ERROR]2018/10/15 04:26:36 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:37.764004011Z [ERROR]2018/10/15 04:26:37 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:38.784005663Z [ERROR]2018/10/15 04:26:38 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:39.804387919Z [ERROR]2018/10/15 04:26:39 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:40.985163744Z [ERROR]2018/10/15 04:26:40 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : j2018-10-15T04:26:41.677984887Z [INFO]2018/10/15 04:26:41 Pulling Image databoxsystems/core-ui-amd64:0.5.0
container-manager       : 2018-10-15T04:26:42.162884236Z [ERROR]2018/10/15 04:26:42 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error
container-manager       : 2018-10-15T04:26:43.286774934Z [ERROR]2018/10/15 04:26:43 ??? L0:[populateDataSources] Error Error writing: internal sercontainer-manager     : ver error

**Comment by Toshbrown** [comment]
There is not enough information for me to help you with this issues.

The title suggests you are building it from source, yet the image names (databoxsystems/core-ui-amd64:0.5.0) suggest your using prebuilt images. 

I need to know:
1)  which commit on the databox repo you are using, 
2) the command used to build databox, 
3) the output of `docker ps` 

To have any chance of knowing what is going on.

**Comment by tsh-box** [comment]
1.Before I build the Databox,
I was clone the databox using => git clone https://github.com/me-box/databox.git

2.And then I run the following commands
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/certs:/certs -v $(pwd)/sdk:/sdk -v -t databoxsystems/databox:0.5.0 /databox start --host-path $(pwd) -sslHostName $(hostname)

3. the output of  docker ps -a
```
CONTAINER ID        IMAGE                                           COMMAND                  CREATED             STATUS                   PORTS                                      NAMES
c92750430890        databoxsystems/driver-app-store-amd64:0.5.0     "./app"                  4 hours ago         Up 4 hours               8080/tcp                                   app-store.1.lx4uguke4ifjxr0zrcpv1tps5
467c98bfff0f        databoxsystems/core-ui-amd64:0.5.0              "./app"                  4 hours ago         Up 4 hours                                                          core-ui.1.lqljx0r71pzixghp9g5hrfneh
997d220b24a8        databoxsystems/core-store-amd64:0.5.0           "/run.sh"                4 hours ago         Up 4 hours               5555-5556/tcp                              app-store-core-store.1.oqwu9nox3gaz81wd3djxt79tg
a76990944e7d        databoxsystems/core-store-amd64:0.5.0           "/run.sh"                4 hours ago         Up 4 hours               5555-5556/tcp                              container-manager-core-store.1.h23eep6nfq980emtj8u6glnxl
b9fe6ca3fc4f        databoxsystems/export-service-amd64:0.5.0       "./service -v"           4 hours ago         Up 4 hours               8080/tcp                                   expo
```

**Comment by tsh-box** [comment]
I have tried by downloading the source code  Zip file from me-box/databox.

And  started with this command ↓↓
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/certs:/certs -v $(pwd)/sdk:/sdk -v -t databoxsystems/databox:0.5.0 /databox start --host-path $(pwd) -sslHostName $(hostname)

Finally I got same errors.

**Comment by Toshbrown** [comment]
If you using this command: 

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/certs:/certs -v $(pwd)/sdk:/sdk -v -t databoxsystems/databox:0.5.0 /databox start --host-path $(pwd) -sslHostName $(hostname)
```

Then it is not building anything and you do not need the code from the git repo. Just running the command on any system (with a correctly configured docker) should start databox. 

Its downloading and starting the images so docker must be working. 

Looks like the container-manager is crashing. 

whats the output of: 
```
docker service ps container-manager

and 

docker service logs container-manager | head -n 50
```

What platform/os/docker version are you on?
 
Try removing the ./certs and ./sdk directories, this can cause errors if upgrading from old versions 

If you are on MacOS make sure the docker demon has access to the folder where you start databox from.

**Comment by tsh-box** [comment]
I have deleted the databox directory,and added new .
and then I start to run databox start command.

>$ docker service ps container-manager
ID                  NAME                      IMAGE                                          NODE                DESIRED STATE       CURRENT STATE             ERROR               PORTS
yfw0982qmgl6        container-manager.1       databoxsystems/container-manager-amd64:0.5.0   DataboxLae          Running             Running 47 seconds ago                        *:443->443/tcp,*:80->80/tcp
otg4f71nkxm6         \_ container-manager.1   databoxsystems/container-manager-amd64:0.5.0   DataboxLae          Shutdown            Shutdown 56 seconds ago


>$  docker service logs container-manager | head -n 50
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:41:51 ContainerManager Started
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:19 ContainerManager Started
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:41:51 Pulling Image databoxsystems/arbiter-amd64:0.5.0
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:19 Starting databox-network
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:19 Pulling Image databoxsystems/core-network-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:41:57 Done pulling Image databoxsystems/arbiter-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:09 Pulling Image databoxsystems/export-service-amd64:0.5.0
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:28 Done pulling Image databoxsystems/core-network-amd64:0.5.0
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:31 Pulling Image databoxsystems/core-network-relay-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:10 Pulling Image databoxsystems/core-store-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:17 Done pulling Image databoxsystems/export-service-amd64:0.5.0
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:38 Done pulling Image databoxsystems/core-network-relay-amd64:0.5.0
container-manager.1.otg4f71nkxm6@DataboxLae    | [INFO]2018/10/16 09:41:41 Restarting the Container Manager
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:18 Done pulling Image databoxsystems/core-store-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:28 /containerManager.go L108:Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:29 Password=N3Hq+Ce8ISmQinD+CRcEUWSZKszWfD1j
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:29 Container Manager Ready and waiting
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:29 Restarting saved apps and drivers
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:29 Installing app-store
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:29 Installing core-ui
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:30 /containerManager.go L159:[GetAllSLAs] failed to get  slaStore. Error getting latest data: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:31 /containerManager.go L159:[GetAllSLAs] failed to get  slaStore. Error getting latest data: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:33 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:34 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:35 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:36 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:39 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:39 Pulling Image databoxsystems/core-ui-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:40 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:42 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:44 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:44 Done pulling Image databoxsystems/core-ui-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:44 Successfully installed core-ui
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:45 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:46 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:47 Pulling Image databoxsystems/driver-app-store-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:50 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [WARNING]2018/10/16 09:42:50 [populateDataSources] GetStoreDataSourceCatalogue tcp://app-store-core-store:5555 500:
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:50 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:51 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:52 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:54 Done pulling Image databoxsystems/driver-app-store-amd64:0.5.0
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:54 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [INFO]2018/10/16 09:42:54 Successfully installed app-store
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:55 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:57 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:42:58 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:00 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:01 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:02 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:03 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:05 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:06 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:07 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:07 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:08 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:10 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:12 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:13 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:14 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:15 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:17 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:18 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:20 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:21 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:22 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:22 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:24 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:26 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:27 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:28 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:29 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:31 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:32 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:33 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:34 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:35 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:36 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:37 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:37 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:38 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:39 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:41 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:42 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:43 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:44 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:46 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:47 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:48 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:49 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:50 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:51 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:53 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:53 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:54 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:55 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:56 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:58 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:43:59 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:00 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:01 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:02 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:04 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:05 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:06 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:07 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:08 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:08 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:10 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:11 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:12 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:13 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:14 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:15 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:16 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:18 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:19 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:20 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:21 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:22 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | [ERROR]2018/10/16 09:44:23 ??? L0:[populateDataSources] Error Error writing: internal server error
container-manager.1.yfw0982qmgl6@DataboxLae    | serviceCreateResponse.Warnings:: []
container-m

**Comment by tsh-box** [comment]
I have run in Linux server,and docker version info 

$ docker version
Client:
 Version:      18.05.0-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   f150324
 Built:        Wed May  9 22:14:54 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.05.0-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   f150324
  Built:        Wed May  9 22:18:36 2018
  OS/Arch:      linux/amd64
  Experimental: false

By the way,I have successfully built the Databox(v0.5.0) in previous days. And I made the another issues (#282,,#286,#288) for app or drivers about v0.5.0.
 But I cannot build it well from this week.

**Comment by Toshbrown** [comment]
The images on docker hub for 0.5.0 have not changed since its release. So if it worked before it should still work. 

try resetting everything:

```
docker rm  $(docker ps -a -q)
docker rmi $(docker images -q) -f
docker volume rm $(docker volume ls -q | grep core-store)
```

**Comment by tsh-box** [comment]
I have solved the internal server error.
Now I could built the Databox and installed app-os-monitor and twitter-sentiment.
app-os-monitor UI is loaded but it looks like not export to https://export.amar.io/.

**Comment by Toshbrown** [comment]
should be fixed now in 0.5.1

```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v -t databoxsystems/databox:0.5.1 /databox start -sslHostName $(hostname)
```
